backend/core/domains/events/models.py

```
@@ -188,6 +240,31 @@ def next_task(self):
+
+        # Method 3: Try to get from original booking session
+        try:
+            from core.domains.bookingflow.models import BookingSession
+            latest_session = BookingSession.objects.filter(
+                booking_data__contains={'event_id': self.id}
+            ).order_by('-created_at').first()
+
+            if latest_session:
+                return latest_session._get_event_duration()
+        except Exception:
+            pass
```

Ruff (0.13.3)

`try`-`except`-`pass` detected, consider logging the exception

(S110)